docs/rest-api/rest-notebook.md

```diff
@@ -93,7 +93,21 @@ limitations under the License.
     </tr>
     <tr>
       <td> sample JSON input </td>
-      <td><pre>{"name": "name of new notebook"}</pre></td>
+      <td><pre>
+{
+  "name": "name of new notebook", 
+  "paragraphs": [
```

Comment:
Maybe we want to describe parameters of json independently.
I'm also curious what's the best way to describe API's parameters in docs. Inner table would be fine? Do you have some other ideas?